release_notes/unreleased.md

```diff
@@ -1 +1,3 @@
 **Unreleased**
+* Upgrade splunk soar app to python 3
```

Comment:
Compatibility changes for Python 3 support

Reply:
done